src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -191,8 +191,8 @@ export default class ExtraproRegistration extends Mixins(DocumentMixin) {
 
   /** Called when this component is mounted. */
   mounted (): void {
-    // set or initialize authorization object
-    this.authorization = this.getContinuationAuthorization || { files: [] } as ContinuationAuthorizationIF
+    this.authorization = this.getContinuationAuthorization ||
+      { files: [], date: null, expiryDate: null } as ContinuationAuthorizationIF
```

Comment:
The reason for the Date and Expiry Date not being synced to the store and thus displayed on the review page is that the watcher on line 270 below didn't fire.

The reason that the watcher didn't fire, even though the "deep watch" option was enabled, is that the Date and Expiry Date were not reactive.

The reason that the Date and Expiry Date were not reactive is that they were undefined in the initial object (only `files` was initialized). Now that I initialize them, they are reactive and the watcher fires and the store is updated and the review step shows their value.

What line 194 does is set the local variable to point (refer) to the same object that is in the store. This is not a new memory allocation; it's the same object. So changing object properties in this component is actually setting those properties in the store object. However, if the store object was initially null then line 195 assigns a new object (memory) to the local variable. This is then synced to the store on line 272.

Reply:
I bet that took some time to identify. Sweet, thanks for the clarification Sev! It makes sense.

Reply:
Nope, I figured out this bug right away.

But then, a while back, I spent time learning (the hard way) about undefined variables and properties 😆